lastpass/assets/logs/lastpass_tests.yaml

```diff
@@ -0,0 +1,331 @@
+      custom:
+        Data: "Length of site passwords"
+        Time: "2024-08-06 05:16:17"
+        evt:
+          name: "Add policy"
+        network:
+          client:
+            geoip: { }
+            ip: "10.10.10.10"
+        policy:
+          name: "Length of site passwords"
+        timestamp: 1.722921377E12
+        usr:
+          name: "abc@test.com"
+      message: |-
+        {
+          "Action" : "Add policy",
+          "Username" : "abc@test.com",
+          "Time" : "2024-08-06 05:16:17",
+          "Data" : "Length of site passwords",
+          "IP_Address" : "10.10.10.10",
+          "timestamp" : 1.722921377E12
+        }
+      tags:
+        - "source:LOGS_SOURCE"
+  - sample: |-
+      {
+          "Time": "2024-08-06 05:16:17",
+          "Username": "abc@test.com",
+          "IP_Address": "10.10.10.10",
+          "Action": "Edit policy",
+          "Data": "Length of site passwords New value: test.com=8",
+          "timestamp": 1722921377000.0  
+      }
+    result:
+      custom:
+        Data: "Length of site passwords New value: test.com=8"
+        Time: "2024-08-06 05:16:17"
+        evt:
+          name: "Edit policy"
+        network:
+          client:
+            geoip: { }
+            ip: "10.10.10.10"
+        policy:
+          name: "Length of site passwords"
+          new_values: "test.com=8"
+        timestamp: 1.722921377E12
+        usr:
+          name: "abc@test.com"
+      message: |-
+        {
+          "Action" : "Edit policy",
+          "Username" : "abc@test.com",
+          "Time" : "2024-08-06 05:16:17",
+          "Data" : "Length of site passwords New value: test.com=8",
+          "IP_Address" : "10.10.10.10",
+          "timestamp" : 1.722921377E12
+        }
+      tags:
+        - "source:LOGS_SOURCE"
+  - sample: |-
+      {
+          "Time": "2024-08-06 05:16:17",
+          "Username": "abc@test.com",
+          "IP_Address": "10.10.10.10",
+          "Action": "Site added",
+          "Data": "VID: 5516541713708306640 from SFolder33",
+          "timestamp": 1722921377000.0
+      }
+    result:
+      custom:
+        Data: "VID: 5516541713708306640 from SFolder33"
+        Time: "2024-08-06 05:16:17"
+        evt:
+          name: "Site added"
+        network:
+          client:
+            geoip: { }
+            ip: "10.10.10.10"
+        shared_folder:
+          name: "SFolder33"
+        site:
+          VID: "5516541713708306640"
+        timestamp: 1.722921377E12
+        usr:
+          name: "abc@test.com"
+      message: |-
+        {
+          "Action" : "Site added",
+          "Username" : "abc@test.com",
+          "Time" : "2024-08-06 05:16:17",
+          "Data" : "VID: 5516541713708306640 from SFolder33",
+          "IP_Address" : "10.10.10.10",
+          "timestamp" : 1.722921377E12
+        }
+      tags:
+        - "source:LOGS_SOURCE"
+
+# The `result` field should be left blank to start. Once you submit your log asset files with
```

Comment:
You can delete these comments as they are just here to help creating the pipelines

Reply:
Comments are deleted.
Thanks